lib-ui-compose/src/commonMain/kotlin/com/ustadmobile/libuicompose/view/report/graphs/Graphs.kt

```diff
@@ -0,0 +1,454 @@
+package com.ustadmobile.libuicompose.view.report.graphs
+
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Text
```

Comment:
This is importing from material(2) - this should be removed. Should import material3 and not use fully qualified names